src/rulesets/operations.ts

```diff
@@ -74,18 +76,17 @@ export const rules = {
     );
   },
   removingOperationId: ({ operations }: SnykApiCheckDsl) => {
-    operations.changed.must(
-      "have consistent operation IDs",
-      (current, next) => {
+    operations.changed
+      .attributes("operationId")
+      .must("have consistent operation IDs", (current, next) => {
```

Comment:
constrain the operation id naming rule to changes to its operationId

**Currently** if you changed changed anything else in the operation it would run this rule 

**Now**  this only runs if the operationId actually changed